hieradata/dev/wso2/wso2am/2.0.0/default/api-key-manager.yaml

```diff
@@ -25,6 +25,10 @@ wso2::apim_traffic_manager:
   username: "%{hiera('wso2::super_admin::username')}"
   password: "%{hiera('wso2::super_admin::password')}"
 
+wso2::template_list:
+  - repository/conf/axis2/axis2_TM.xml
+  - repository/conf/registry_TM.xml
+
```

Comment:
We don't need to define these two templates. These templates will be referred in embedded files.